src/models/deck.ts

```diff
@@ -21,6 +23,36 @@ export interface Deck {
   cards: Card[]; // not populated until individual cards fetched and put onto this object
 }
 
+export function deckToJson(deck: Deck) {
+  return {
+    title: deck.metaData.title,
+    description: deck.metaData.desc,
+    access: 'Public',
+    default_flang: deck.metaData.frontLang,
+    default_blang: deck.metaData.backLang,
+    userId: 'ad4c76a0-8e0a-4518-b055-5d1dc3ebc4f0', // Todo: replace with id/token
```

Comment:
this is legacy and can be removed, the backend knows the user since each request is authorized